.circleci/config.yml

```diff
@@ -54,21 +58,21 @@ jobs:
       - run: mkdir -p test_volume && chmod -R a+rw test_volume
       - run: ./foreman/run_tests.sh
 
-      # Run Illumna tests.
-      - run: sudo chown -R circleci:circleci workers/test_volume/
-      - run: .circleci/filter_tests.sh -t illumina
-
-        # Run Smasher tests.
+      # Run Smasher tests.
       - run: sudo chown -R circleci:circleci workers/test_volume/
       - run:
           command: .circleci/filter_tests.sh -t smasher
           # Smashing can apparently take a long time on CI
           no_output_timeout: 36000
-
-        # Run API Tests.
+      
+      # Run API Tests.
       - run: sudo chown -R circleci:circleci workers/test_volume/
       - run: ./api/run_tests.sh
 
+      # Run Illumna tests.
+      - run: sudo chown -R circleci:circleci workers/test_volume/
+      - run: .circleci/filter_tests.sh -t illumina
```

Comment:
🎉

Reply:
Wait I'm confused, I know you already removed one extra instance of these, but before you did so there were actually 3 of them so at this point we're down to 2 instead of 1.